sky/templates/aws-ray.yml.j2

```diff
@@ -187,10 +187,9 @@ setup_commands:
   # Line 'mkdir -p ..': disable host key check
   # Line 'python3 -c ..': patch the buggy ray files and enable `-o allow_other` option for `goofys`
   - mkdir -p ~/.ssh; touch ~/.ssh/config;
-    pip3 --version > /dev/null 2>&1 || (curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && echo "PATH=$HOME/.local/bin:$PATH" >> ~/.bashrc);
```

Comment:
Is this removed because installing + activating conda into `(base)` ensures `pip3` is installed? May want to update L182 to say that. (Dug up https://github.com/skypilot-org/skypilot/commit/bfa7eab6911f3a450d020658d141fc04323b87e1 - we added pip3 because custom images may not have it.)

Reply:
Yes, with the conda installed and the base env activated, the pip should exist.

Reply:
Fixed the comment in the yaml templates. Thanks!